libraries/AP_InertialSensor/AP_InertialSensor_BMI055.cpp

```diff
@@ -256,7 +256,7 @@ void AP_InertialSensor_BMI055::read_fifo_accel(void)
         return;
     }
     // data is 12 bits with 16g range, 7.81mg/LSB
-    const float scale = 7.81 * 0.001 * GRAVITY_MSS / 16.0;
+    const float scale = 7.81 * 0.001 * GRAVITY_MSS / 16.0f;
```

Comment:
Why did just the 16.0f constant gain the specifier here?

Reply:
Because I missed the other two.